Mailbox/Person.cs

```diff
@@ -1,4 +1,22 @@
-﻿namespace Mailbox
-{
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
 
+namespace Mailbox {
+    public struct Person : IEquatable<Person> {
+        public string _FirstName { get; }
+        public string _LastName { get; }
+
+        public Person(string _FirstName, string _LastName) {
+            this._FirstName = _FirstName;
```

Comment:
```suggestion
            FirstName = firstName;
```
Parameters should be named with camelCase

Reply:
You're right. I don't remember what I was doing, but I swear I had a reason!

Thanks.